packages/validator/README.md

```diff
@@ -0,0 +1,24 @@
+<!---
+
+  Copyright 2019 Google Inc. All Rights Reserved.
+ 
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ 
+       http://www.apache.org/licenses/LICENSE-2.0
+ 
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+# Bubblewrap Validator
+
+Bubblewrap Validator is a JavaScript library that helps developers to verify if their
```

Comment:
The idea is that this will validate agains whatever quality criteria we have. Currently this involves Lighthouse, but might be different in the future. I'm wondering if it's worth documenting the quality criteria here too, but that will mean another place to change if it changes.

Reply:
Where is the canonical place for the quality criteria? We should at least link to it.

Reply:
Added alink